aws/policy/compute.yaml

```
@@ -174,17 +182,13 @@ Statement:
       - elasticloadbalancing:SetLoadBalancerPoliciesOfListener
       - elasticloadbalancing:SetSecurityGroups
       - elasticloadbalancing:SetWebACL
-      - elasticfilesystem:CreateFileSystem
-      - elasticfilesystem:CreateMountTarget
-      - elasticfilesystem:CreateTags
-      - elasticfilesystem:DeleteFileSystem
-      - elasticfilesystem:DeleteMountTarget
-      - elasticfilesystem:Describe*
-      - elasticfilesystem:ListTagsForResource
-      - elasticfilesystem:PutLifecycleConfiguration
-      - elasticfilesystem:TagResource
-      - elasticfilesystem:UntagResource
-      - elasticfilesystem:UpdateFileSystem
+      - elasticloadbalancing:DescribeInstanceHealth
```

```suggestion
      - elasticloadbalancing:Describe*
```
To save space in the policies files we prefer to use wildcards on Describe conditions, this way all possible describe conditions are covered by one line